src/test/java/org/agileandbeyond/dandd/tddexercise/RaceTest.java

```diff
@@ -0,0 +1,27 @@
+package org.agileandbeyond.dandd.tddexercise;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class RaceTest {
+
+	@Before
+	public void setUp() {
+		
+	}
+	
+	@Test
+	public void getAbilityModifierForOrcStrengthShouldBe2() {
+		Race race = new Race();
+		
+		race.setRaceType(RaceType.ORC);
+		assertEquals(race.getAbilityModifier(AbilityType.STRENGTH), 2);		
+	}
+	
+	public void getAbilityModifierForDwarfCharismaShouldBeNegative1() {
```

Comment:
Need to add the @Test annotation to this test otherwise it will not be executed.